### ISSUE TITLE
[Feature Request] Implement LSP Crossover LeftRight x8

### ISSUE THREAD

**Issue by p-chan5** [wwmm/easyeffects#2770]
Hi guys! I've been testing this plugin for a couple of weeks and I think it would be amazing to include it in EasyEffects!

This plugin allows you to split the signal using crossover filters. In addition to inverting the phase and adding delay to each of these bands, similar to [Delay Compensator x2 Stereo](https://lsp-plug.in/?page=manuals&section=comp_delay_x2_stereo) but much more specific. That's why the [LeftRight](https://lsp-plug.in/?page=manuals&section=crossover_lr) version makes sense, because you have to apply delay to one of the two channels to really hear the effect.

What do you think?

**Comment by wwmm** [comment]
>What do you think?

I do not think it is viable in a pipeline that is restricted to 2 channels. Even if we somehow allow its 2 output channels to be selected by the user you will always loose whatever it is going through the other 6 output channels.

**Comment by p-chan5** [comment]
> Even if we somehow allow its 2 output channels to be selected by the user you will always loose whatever it is going through the other 6 output channels.

Yeah, I forgot to mention that... The remaining output channels allow you to send those band-split parts of the signal to more sources. Although the plugin only supports `input_1` and `input_2`, the signal can be processed only by `output_1` and `output_2`, without any losses.

This is the introduction video of the plugin, showing a brief example of how these outputs work: [LSP Crossover Plugin Series](https://www.youtube.com/watch?v=g8cShrKtmKo)

**Comment by wwmm** [comment]
>the signal can be processed only by output_1 and output_2, without any losses

I imagine that somehow the plugin is summing all the other bands outputs and sending the reconstructed signal to the outputs 1 and 2. But if we use the monitor channels the main feature people would expect from a crossover that is being able to apply different effects for each band won't work. What is the point then? Just being able to apply different delay values for each frequency region? I am not sure about when this would be useful without additional processing of each band output before the reconstruction. It seems a ultra specific use case.

**Comment by p-chan5** [comment]
> I imagine that somehow the plugin is summing all the other bands outputs and sending the reconstructed signal to the outputs 1 and 2.

If the processing is quite specific then yes, you would have to connect both outputs separately instead of just using the master channel.

> What is the point then? Just being able to apply different delay values for each frequency region?

It could also work as an equalizer with crossover filters. Let's say you split the signal into bass, midrange and treble. You can cut/boost any of these bands and, better yet, add delay to the mids and highs of the right channel.

![0001](https://github.com/wwmm/easyeffects/assets/123143444/dddedd32-98dc-48ea-aeb3-52475bfb4bc0)

**Comment by wwmm** [comment]
>It could also work as an equalizer with crossover filters. Let's say you split the signal into bass, midrange and treble. You can cut/boost any of these bands and, better yet, add delay to the mids and highs of the right channel.

Yes. But with the exception of the delay this frequency response can probably be achieved with the equalizer. I think that the question we have to answer is what is the advantage of applying the delay between these bands. It is not hard to think about a application of the delay between channels. But I am not sure about what we will be able to do applying delay between these bands.

**Comment by p-chan5** [comment]
> Yes. But with the exception of the delay this frequency response can probably be achieved with the equalizer.

So far I have not been able to replicate the same frequency response with the LSP equalizer, I get "similar" results but nothing close to what the Linear Phase mode sounds like.

> But I am not sure about what we will be able to do applying delay between these bands.

I'm still dealing with it... but I would like to create a delay in the high frequencies of both channels that sounds amazing!

Could you think about it? Maybe one day it will be useful to many users.

**Comment by wwmm** [comment]
>Could you think about it? Maybe one day it will be useful to many users.

Let's keep this open for now. I will try to take a look at the possibilities when I have some time.
